fix(installer): duplicate port in suggested board URL

arcalys opened the pull request:
### Fixed

- Duplicate non standard port when port is already present in `$_SERVER['HTTP_HOST']`.

### Example

![image](https://github.com/user-attachments/assets/5244cea2-f87f-4653-a38c-5f5fb2ec6187)

Commit:
fix(installer): duplicate port in suggested board URL

dvz commented:
Thanks; can we test for `:` alone? We can also use `str_contains()`.

The 3 conditions be can now split into separate lines, as well.

arcalys commented:
> Thanks; can we test for `:` alone? We can also use `str_contains()`.
> 
> The 3 conditions be can now split into separate lines, as well.

Updated

dvz committed:
Revert extra change
